build.gradle

```diff
@@ -441,6 +441,44 @@ task bundle(type: (isWindows()) ? Zip : Tar) {
     }
 }
 
+task bundleRestricted(type: Tar) {
+    dependsOn('crossDistPlatformLibs')
+    dependsOn('crossDist')
+    dependsOn('distSources')
+    def simpleOsName = HostManager.simpleOsName()
+    baseName = "kotlin-native-$simpleOsName-$konanVersionFull-restricted"
+    from("$project.rootDir/dist") {
+        include '**'
+        exclude 'dependencies'
+        exclude 'klib/testLibrary'
+        exclude 'dist/klib/platform/macos_x64'
```

Comment:
Shouldn't these paths be relative to `dist` like the path above? I mean `klib/platform/macos_x64` instead of `dist/klib/platform/macos_x64`.